docs/LISTING-README-TEMPLATE.md

```diff
@@ -0,0 +1,97 @@
+# Description
+**Focus on what this software is and the VALUE it brings to users.**
+------
+(replace all text in italics.)
+
+_Write a few paragraphs about *WHAT* the listing is, *WHAT* the software does, and the *VALUE* it brings to users._
+
+_Include specific details on what this 1-click provides, and the value to users. Enumerate the values this 1-click's config gives the user. Enumerate the value this software combination, specifically for stacks containing multiple pieces of software, bring to the user. 
+
+_Include links for the user to learn more about the software that are not included in the software included section._
+
+_Recommend CPU and Memory minimum requirements and node pools for test/dev and production for X amount of user_
+
+Thank you to all the contributors whose hard work make this software valuable for users.
```

Comment:
is this informational and also to be deleted (I think so). What about blockquoting this as well as italics and a little intro 

_A note from the DigitalOcean Sharks:
> Thank you etc_

Reply:
The "thank you to contributors" should be included for all OpenSource software. I'll add an intro to this line as it is proceeded by a lot of italics.

Reply:
Updated.